### ISSUE TITLE
Some files of the sscbench-kitti360 dataset are missing

### ISSUE THREAD

**Issue by daifan2000** [ai4ce/SSCBench#23]
Hi! Thank you for your excellent work. I downloaded the SSCBench-KITTI360 dataset from the Hugging Face link you provided. After unzipping it, I found that some files were missing. There are only 8,483 scenes in the voxels folder of the training set, which is four short. It seems that there are four missing files in 2013_05_28_drive_0003_sync. There are only 2,165 scenes in the test set, but you said there were 2,566 scenes.
    Looking forward to your reply, thank you!

**Comment by Louis-Leee** [comment]
Thanks for your interest in our work. I am sorry for the confusion. Here is the clarification for your questions:

1) in the "voxels" folder, after careful double-checking, we believe there should be 8483 frames in our final version. You can delete files from "000970_1_1.npy" to  "000985_1_8.npy"  in preprocess/labels/2013_05_28_drive_0003_sync. In fact, these extra data will not be loaded by the MonoScene dataloader.

2) Yes, the test set contains only 2165 frames, We have made some optimizations and adjustments to the data to ensure better quality.

we will upload the modified version to arxiv. If you have any more questions, please let me know.

Best, 

I will close this issue.
